### PR TITLE
add enhanced feedings statistics

### PR DESCRIPTION
**Pull request by BenjaminHae**
Add statistics for the last 3 days / 2 weeks
easily configurable

### CODE REVIEW

**Comment by BenjaminHae** [comment]
I'm not sure, where the tests for this part are located.

File: dashboard/templatetags/cards.py [hunk 1]
```diff
@@ -235,22 +238,40 @@ def _feeding_statistics(child):
...
+            }, 
+            {
+                'start': timezone.now()-timezone.timedelta(weeks=2), 
+                'title': _('Feeding frequency') + " 2 "+_("weeks")
+            },
+            {
+                'start': timezone.make_aware(datetime.combine(date.min, time(0,0)) + timezone.timedelta(days=1) ),
+                'title': _('Feeding frequency')
+            }
+        ]
```
Comment (cdubz) [inline dashboard/templatetags/cards.py]: There is no need to concat the translated strings here. "Feeding frequency" is already a translation string, but "days" and "weeks" are not. Anyway, I'd prefer to have these strings as "Feeding frequency (past 3 days)" and "Feeding frequency (past 2 weeks)".

**Comment by BenjaminHae** [comment]
Thank you for your input. Took me some time to setup gulp, but now I like it :)

I think I've fixed everything.

**Comment by cdubz** [comment]
Yeah gulp is a bit of beast :hurtrealbad:

Thanks again for the contribution!

**Pull request merged**